README.md

```diff
@@ -188,16 +188,23 @@ Here is a snippet of a `Dockerfile` to add in tini (be sure to use it in `CMD` o
 
 ```Dockerfile
 # grab tini for signal processing and zombie killing
-ENV TINI_VERSION v0.9.0
-RUN set -x \
-	&& curl -fSL "https://github.com/krallin/tini/releases/download/$TINI_VERSION/tini" -o /usr/local/bin/tini \
-	&& curl -fSL "https://github.com/krallin/tini/releases/download/$TINI_VERSION/tini.asc" -o /usr/local/bin/tini.asc \
+ENV TINI_VERSION v0.18.0
+RUN set -ex \
+	&& curl -fSL --compressed "https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-static" -o /usr/local/bin/tini \
+	&& curl -fSL --compressed "https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-static.asc" -o /usr/local/bin/tini.asc \
 	&& export GNUPGHOME="$(mktemp -d)" \
-	&& gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys 6380DC428747F6C393FEACA59A84159D7001A4E5 \
+	&& for key in \
+				595E85A6B1B4779EA4DAAEC70B588DFF0527A9B7 ; \
+			do \
+				gpg --batch --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
+				gpg --batch --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys "$key" || \
+				gpg --batch --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" || \
+				gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys "$key" ; \
```

Comment:
This keyserver looping (and looping over a single key) is unnecessary for the official images build infrastructure and definitely not something we'd recommend (see https://github.com/docker-library/official-images/issues/4252#issuecomment-405749749, https://github.com/docker-library/php/pull/666, https://github.com/tianon/pgp-happy-eyeballs).